admin_guide/manage_scc.adoc

```diff
@@ -143,10 +147,26 @@ $ oc edit scc <scc_name>
 
 == Updating the default Security Context Constraints
 
-If you would like to reset your security context constraints to the default settings for any reason
-you may delete the existing security context constraints and restart your master.  The default
-security context constraints will only be recreated if no security context constraints exist in the
-system.
+Default SCCs will be created when the master is started if there are no existing
+SCCs already in the system.  If you would like to reset your SCCs to the defaults
```

Comment:
Default SCCs will be created when the master is started if they are missing. To reset SCCs to defaults, or update existing SCCs to new default definitions after an upgrade, ...